python/tox.ini

```diff
@@ -25,6 +25,33 @@ deps =
 changedir = {[tox]setupdir}
 commands = flake8
 
+[flake8]
+# TODO(PROTON-2322) decrease the limit
+max-line-length = 150
+
+# TODO(PROTON-2322) re-enable all these warnings
+ignore =
+    # do not use bare 'except'
+    E722,
+    # imported but unused
+    F401,
+    # 'from proton import *' used; unable to detect undefined names
+    F403,
```

Comment:
It may not be worth adding this test until this list is much smaller!

Reply:
it's small compared to what dispatch has; and I think it is worth it, just to preserve the style fixes already in.

I am thinking, is flake8 running against the test code, currently? Afaik with this setting it doesn't, it only checks what's installed.

Reply:
I'm not sure I understand the question. But tox runs tests against a build. It does it by building a package and then installing it first, but then runs tests etc.

Reply:
I mean, flake8 is a code linter. It makes sense IMO to run it against test code as well as production code.

Reply:
I see, that's true. I don't think it'd be worth complicated the cmake build for this at the moment though. Maybe in a CI job

Reply:
Ok, I'll consider doing a separate linting job in CI.

Reply:
https://issues.apache.org/jira/browse/PROTON-2336